### DISCUSSION TITLE
How we can add or create game in Resource?

### DISCUSSION THREAD

**Discussion by sachinb80** [Sunbird-inQuiry/Community#68] [Q&A] [answered]
I would need to understand how this part works.
How to add a game in Resource?
![image](https://user-images.githubusercontent.com/109504832/227458225-d899e00a-7ade-48dc-ad92-8ca4cb2cf123.png)
![image](https://user-images.githubusercontent.com/109504832/227458383-26b822e4-08f3-4a99-95c2-eb2e181166d8.png)

**Comment by vaibhavbhuva** [comment, marked as answer]
@sachinb80 This is not related to inQuiry. So please raise the request to [Knowlg Community](https://github.com/orgs/Sunbird-Knowlg/discussions).

cc: @PoojaJManjooran

**Comment by sachinb80** [comment]
sure.

Warm Regards,
Sachin Baviskar
QA
Tekdi Technologies Private Limited.
+91 9511701797

**Reply by vaibhavbhuva** [reply]
@sachinb80 is your issue resolved? If yes then can you please mark this thread as answered? If not then please follow up with relevant details.

cc: @sajeshkayyath @PoojaJManjooran

**Reply by sachinb80** [reply]
sure
